app/javascript/src/components/Dashboard/Notes/utils.js

```diff
@@ -0,0 +1,5 @@
+import dayjs from "dayjs";
+
+export const dateTimeFormat = date => dayjs(date).format("dddd, hh:mm A");
+
+export const relativeCreationTime = date => dayjs(date).fromNow();
```

Comment:
What is `CreationTime` in the context of a `utils` file? Keep the utils name generic and sensible.

Reply:
I will make the necessary changes. I believe `formatWithRelativeTime` would be better.

Reply:
I went through your Slack thread again, to check if I missed out something.

I saw that you've already suggested a name for the function: `formatDateRelativeToNow`. I will stick on to this.

Sorry for the trouble!